src/Symfony/Component/ErrorHandler/Tests/Fixtures/InterfaceWithAnnotatedParameters.php

```diff
@@ -13,14 +13,14 @@ interface InterfaceWithAnnotatedParameters
     public function whereAmI();
 
     /**
-     * @param       $noType
+     * @param       $noType with $dollar after
```

Comment:
Shouldn't we keep `@param $noType` as an extra tested case?

Reply:
there's already `@param $defined` L20

Reply:
I saw it but having more than one space between `@param` and the var is a different case that was tested before and not anymore :man_shrugging: I let you decide if it's worth adding it back.